File: exercicios/1035/1035.py
vl = input().split()

a = int(vl[0])
b = int(vl[1])
c = int(vl[2])
d = int(vl[3])

if(b > c) and (d > a) and (c + d > a + b) and (c > 0) and (d > 0):
    print("Valores Aceitos")
else: print("Valores não aceitos")

File: exercicios/R/1035/1035.r
input = file('stdin','r')

line = strsplit(readLines(input, n=1), " ")

a = as.integer(line[[1]][1])
b = as.integer(line[[1]][2])
c = as.integer(line[[1]][3])
d = as.integer(line[[1]][4])

if(b > c & d > a & (c + d) > (a + b) & c > 0 & d > 0){
        write(sprintf("Valores aceitos"),'')
}else{ write(sprintf("Valores nao aceitos"),'')}
